template/.eslintrc

```diff
@@ -31,12 +30,12 @@
       {
         "capIsNewExceptions": [
           "List",
-          "Map",
-          "Immutable"
+          "Map"
         ]
       }
     ],
     "react-hooks/rules-of-hooks": "error",
+    "react-hooks/exhaustive-deps": "warn",
```

Comment:
Isn't this warn by default? Btw isnt the `/rules-of-hooks` error by default too? Do we need to specify this here?